examples/games/breakout.rs

```diff
@@ -72,8 +72,22 @@ fn main() {
         // Configure how frequently our gameplay systems are run
         .insert_resource(FixedTime::new_from_secs(TIME_STEP))
         .add_system(update_scoreboard)
-        .add_system(bevy::window::close_on_esc)
-        .run();
+        .add_system(bevy::window::close_on_esc);
+
+    // stepping is only implemented in the single-threaded executor for the
+    // moment, so let's force Main and FixedUpdate to use it.
+    // XXX temporary, for proof-of-concept
+    app.get_schedule_mut(CoreSchedule::FixedUpdate)
+        .expect("CoreSchedule::FixedUpdate to be present in app")
+        .set_executor_kind(bevy::ecs::schedule::ExecutorKind::SingleThreaded);
+    app.get_schedule_mut(CoreSchedule::Main)
+        .expect("CoreSchedule::Main to be present in app")
+        .set_executor_kind(bevy::ecs::schedule::ExecutorKind::SingleThreaded);
+    app.add_startup_system(setup_stepping)
+        .add_system(stepping_input.ignore_stepping())
+        .add_system(stepping_ui.ignore_stepping());
```

Comment:
I made this change to breakout as a proof-of-concept for stepping, but should it remain in this example, or should stepping have a dedicated example?

Reply:
Hmm. Fundamentally we need something complex enough to show off the feature. I think that this is a good enough choice for now.